.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build and test
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11", "3.12", "3.13"]
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Need to clone everything for the git tags.
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+
+      - name: Install prereqs for setuptools
+        run: pip install wheel
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip uv
+          uv pip install --system -r requirements.txt
+
+      - name: Install pytest packages
+        run: uv pip install --system pytest pytest-xdist pytest-cov
+
+      - name: List installed packages
+        run: uv pip list
+
+      - name: Build and install
+        run: uv pip install --system --no-deps -e .
+
+      - name: Run tests
+        run: |
+          sdm-tools --help
```

Comment:
when are tests going to appear?

Reply:
I am not sure, but I was not planning on adding any for the initial version.

Reply:
You added a test but you still need to run the tests here.